project/NativeImage.scala

```diff
@@ -314,7 +318,7 @@ object NativeImage {
         sourcesDiff: ChangeReport[File] =>
           if (sourcesDiff.modified.nonEmpty)
             rebuild(s"Native Image is not up to date")
```

Comment:
also redundant string prefix here, although i guess it doesn't matter

Reply:
Also fixed https://github.com/enso-org/enso/pull/10374#discussion_r1675336813